dataplane/cmd/sdxinternal.go

```diff
@@ -18,7 +18,7 @@ func init() {
 				Usage:       "creates a new SDX cluster",
 				Description: `basic SDX cluster creation`,
 				Before:      cf.CheckConfigAndCommandFlagsWithoutWorkspace,
-				Flags:       fl.NewFlagBuilder().AddResourceDefaultFlags().AddFlags(fl.FlEnvironmentName, fl.FlInputJson, fl.FlWithExternalDatabaseOptional, fl.FlWithoutExternalDatabaseOptional, fl.FlCloudStorageBaseLocationOptional, fl.FlCloudStorageInstanceProfileOptional).AddAuthenticationFlags().Build(),
+				Flags:       fl.NewFlagBuilder().AddResourceDefaultFlags().AddFlags(fl.FlEnvironmentName, fl.FlInputJson, fl.FlWithExternalDatabaseOptional, fl.FlWithoutExternalDatabaseOptional, fl.FlRuntimeOptional, fl.FlCloudStorageBaseLocationOptional, fl.FlCloudStorageInstanceProfileOptional).AddAuthenticationFlags().Build(),
 				Action:      sdx.CreateSdx,
 				BashComplete: func(c *cli.Context) {
 					for _, f := range fl.NewFlagBuilder().AddResourceFlagsWithOptionalName().AddFlags(fl.FlEnvironmentName, fl.FlEnvironmentName, fl.FlInputJson, fl.FlWithExternalDatabaseOptional, fl.FlWithoutExternalDatabaseOptional, fl.FlCloudStorageBaseLocationOptional, fl.FlCloudStorageInstanceProfileOptional).AddAuthenticationFlags().Build() {
```

Comment:
You should add the new flag to the 'bashcomplete' too.

Reply:
Thanks, done!